cases/function/window/test_window_union.yaml

```diff
@@ -1668,10 +1851,6 @@ cases:
           2, 100, 111, 5
           3, 101, 111, 0
           4, 102, 111, 0
-          5, 0,   114, 9
```

Comment:
same here

Reply:
This expectation is correct. It may be that copying and pasting causes diff display problems